staging/src/k8s.io/controller-manager/app/client_builder.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package controller
+package app
```

Comment:
I would prefer to see the client builders put into their own package like `k8s.io/controller-manager/pkg/clientbuilder`. I think the "app" package is less relevant for controller-manager since this is shared library as opposed to a component.

Reply:
Thanks. Updated to `k8s.io/controller-manager/pkg/clientbuilder`. 😊